kyuubi-hive-jdbc/src/main/java/org/apache/kyuubi/jdbc/hive/KyuubiConnection.java

```diff
@@ -856,13 +857,24 @@ private String getSessionValue(String varName, String varDefault) {
     return varValue;
   }
 
-  // copy loginTimeout from driver manager. Thrift timeout needs to be in millis
-  private void setupLoginTimeout() {
-    long timeOut = TimeUnit.SECONDS.toMillis(DriverManager.getLoginTimeout());
```

Comment:
Do we need to consider the scenario where only `loginTimeout` is set, but `connectTimeout` and `socketTimeout` are not set?
It looks like this change is not intended to use JDBC's loginTimeout.

Reply:
It is based on the discussion https://github.com/apache/hive/pull/3379#discussion_r905980870, do you have different ideas?

Reply:
After reading the discussion on several PRs, this change looks good to me.
[HIVE-12371](https://issues.apache.org/jira/browse/HIVE-12371)
[HIVE-22196](https://issues.apache.org/jira/browse/HIVE-22196)
[HIVE-26336](https://issues.apache.org/jira/browse/HIVE-26336)